models/article-models.js

```diff
@@ -0,0 +1,35 @@
+const db = require("../db/connection");
+
+exports.fetchArticle = (articleId) => {
+  return db.query("SELECT * FROM articles WHERE article_id = $1;", [articleId]).then((results) => {
+    if (results.rows.length < 1) {
+      return Promise.reject({ status: 404, msg: "Not Found" });
+    } else {
+      return results.rows[0];
+    }
+  });
+};
+
+exports.updateArticle = (articleId, votes) => {
+  //console.log("MODEL");
+  //console.log(articleId, votes, "MODEL ARTICLE ID AND VOTES");
+  if (!votes.inc_votes) {
+    return Promise.reject({ status: 400, msg: "Bad Request - incorrect body format" });
+  }
+  return db.query("SELECT votes FROM articles WHERE article_id = $1;", [articleId.article_id]).then((results) => {
```

Comment:
You could use the function you created for get article/:article_id to check this instead of writing out this whole query again to keep your code more DRY